<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>630</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>850</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Create or Update a Client</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="clientForm">
      <property name="title">
       <string>Client Form</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0" colspan="2">
        <widget class="QFrame" name="data">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="labelName">
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="name">
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>Juan Lorente</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelEmail">
            <property name="text">
             <string>Email</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="email">
            <property name="placeholderText">
             <string>juanlorente@hotmail.com</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelPhoneNumber">
            <property name="text">
             <string>Phone Number</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="phone_number">
            <property name="text">
             <string>+34</string>
            </property>
            <property name="placeholderText">
             <string/>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelAddress">
            <property name="text">
             <string>Address</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="address">
            <property name="placeholderText">
             <string>Calle Portugal 12</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="labelCity">
            <property name="text">
             <string>City</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="city">
            <property name="placeholderText">
             <string>Monda</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="labelRegion">
            <property name="text">
             <string>Region</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="region">
            <property name="placeholderText">
             <string>MÃ¡laga</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="labelPostalCode">
            <property name="text">
             <string>Postal Code</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="postal_code">
            <property name="placeholderText">
             <string>29110</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="labelNote">
            <property name="text">
             <string>Note</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLineEdit" name="note">
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>Difficult to get to the house, call first, be careful for dogs</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QGroupBox" name="additionalData">
         <property name="title">
          <string>Additional Information</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QFrame" name="addiData">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QFormLayout" name="formLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="labekFirstVisit">
               <property name="text">
                <string>Membership Date</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QDateEdit" name="first_visit">
               <property name="date">
                <date>
                 <year>2020</year>
                 <month>1</month>
                 <day>3</day>
                </date>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="labelWaterOnly">
               <property name="text">
                <string>Water Only</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QCheckBox" name="water_only">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="labelRentalPrice">
               <property name="text">
                <string>Rental Price</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="labelWaterPrice">
               <property name="text">
                <string>Water Price</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="labelMachineType">
               <property name="text">
                <string>Machine Type</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLineEdit" name="machine_type">
               <property name="text">
                <string>Elegance</string>
               </property>
               <property name="clearButtonEnabled">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="labelMachinesInPos">
               <property name="text">
                <string>Machines In Possession</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QSpinBox" name="machines_in_pos">
               <property name="value">
                <number>1</number>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="labelBottlesInPos">
               <property name="text">
                <string>Bottles In Possession</string>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QSpinBox" name="bottles_in_pos"/>
             </item>
             <item row="7" column="0">
              <widget class="QLabel" name="labelMethodContact">
               <property name="text">
                <string>Method Of First Contact</string>
               </property>
              </widget>
             </item>
             <item row="7" column="1">
              <widget class="QLineEdit" name="method_contact">
               <property name="text">
                <string/>
               </property>
               <property name="placeholderText">
                <string>Website, Email, Facebook...</string>
               </property>
               <property name="clearButtonEnabled">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="8" column="0">
              <widget class="QLabel" name="labelLang">
               <property name="text">
                <string>Language</string>
               </property>
              </widget>
             </item>
             <item row="8" column="1">
              <widget class="QLineEdit" name="lang">
               <property name="text">
                <string>English</string>
               </property>
               <property name="clearButtonEnabled">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QDoubleSpinBox" name="rental_price">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="prefix">
                <string/>
               </property>
               <property name="suffix">
                <string/>
               </property>
               <property name="maximum">
                <double>200.000000000000000</double>
               </property>
               <property name="value">
                <double>121.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QDoubleSpinBox" name="water_price">
               <property name="maximum">
                <double>50.000000000000000</double>
               </property>
               <property name="value">
                <double>7.150000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="account">
         <property name="minimumSize">
          <size>
           <width>350</width>
           <height>0</height>
          </size>
         </property>
         <property name="title">
          <string>Account</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="0">
           <widget class="QFrame" name="accountData">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QFormLayout" name="formLayout_3">
             <item row="1" column="0">
              <widget class="QLabel" name="labelContactPerson">
               <property name="text">
                <string>Contact Person</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="contact_person">
               <property name="clearButtonEnabled">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QGroupBox" name="billing">
            <property name="title">
             <string>Billing</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="0" column="0">
              <widget class="QFrame" name="billingData">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QFormLayout" name="formLayout_4">
                <item row="0" column="0">
                 <widget class="QLabel" name="labelBillAddress">
                  <property name="text">
                   <string>Address</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="labelBillCity">
                  <property name="text">
                   <string>City</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="labelBillRegion">
                  <property name="text">
                   <string>Region</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="labelBillPostalCode">
                  <property name="text">
                   <string>Postal Code</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLineEdit" name="bill_address">
                  <property name="clearButtonEnabled">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLineEdit" name="bill_city">
                  <property name="clearButtonEnabled">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLineEdit" name="bill_region">
                  <property name="clearButtonEnabled">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QLineEdit" name="bill_postal_code">
                  <property name="clearButtonEnabled">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="btnRegister">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Register</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>name</tabstop>
  <tabstop>email</tabstop>
  <tabstop>phone_number</tabstop>
  <tabstop>address</tabstop>
  <tabstop>city</tabstop>
  <tabstop>region</tabstop>
  <tabstop>postal_code</tabstop>
  <tabstop>note</tabstop>
  <tabstop>account</tabstop>
  <tabstop>bill_address</tabstop>
  <tabstop>bill_city</tabstop>
  <tabstop>bill_region</tabstop>
  <tabstop>bill_postal_code</tabstop>
  <tabstop>contact_person</tabstop>
  <tabstop>first_visit</tabstop>
  <tabstop>water_only</tabstop>
  <tabstop>machine_type</tabstop>
  <tabstop>machines_in_pos</tabstop>
  <tabstop>bottles_in_pos</tabstop>
  <tabstop>method_contact</tabstop>
  <tabstop>lang</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
